public/data/aalmeida.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Andre Almeida",
+  "type": "personal",
+  "bio": "Software Engineer and Frontend Specialist!",
+  "avatar": "https://github.com/aalmeida00.png",
+  "links": [
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/aalmeida00",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aalmeida00",
+      "icon": "twitter"
+    },
+    {
+      "name": "Find me at LinkedIn",
+      "url": "https://www.linkedin.com/in/andre-almeida1/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Check it out my blog",
+      "url": "https://aalmeida.tech/",
+      "icon": "blog"
+    }
+  ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```